# Purge Duplicate Contigs

Purge contigs marked as duplicates by purge_dups (could be haplotypic duplication or overlap duplication). The contigs are purged from the first assembly (hap1, pri...), added to the second assembly (hp2, alt... ), then the 2nd assembly is purged as well. If you think only one of the assemblies needs purging, use the VGP6b workflow. 
This workflow is the 6th workflow of the VGP pipeline. It is meant to be run after one of the contigging steps (Workflow 3, 4, or 5)

## Inputs

1. Hifi long reads - trimmed [fastq] (Generated by Cutadapt in the contigging workflow)
2. Primary Assembly (hap1) [fasta] (Generated by the contigging workflow)
3. Alternate Assembly (hap2) [fasta] (Generated by the contigging workflow)
4. K-mer database [meryldb]  (Generated by the k-mer profiling workflow)
5. Genomescope model parameters [txt] (Generated by the k-mer profiling workflow)
6. Estimated Genome Size [txt]
7. Database for busco lineage (recommended: latest) 
8. Lineage of your species for Busco Orthologs (recommended: vertebrata)
9. Name of first haplotype
10. Name of second haplotype


## Outputs

1. Haplotype 1 purged assembly (Fasta and gfa)
2. Haplotype 2 purged assembly (Fasta and gfa)
3. QC: BUSCO report for both assemblies
4. QC: Compleasm report for both assemblies
5. QC: Merqury report for both assemblies
6. QC: Assembly statistics for both assemblies
7. QC: Nx plot for both assemblies
8. QC: Size plot for both assemblies
